app/static/js/eventList.js

```diff
@@ -94,7 +94,7 @@ function updateIndicatorCounts(isChecked){
       $("#viewPastEventsToggle").prop(toggleStatus, true);
 
       // use ternary operators to populate the tab with a number if there are events, and clear the count if there are none
-      studentLedEventsCount > 0 ? $("#studentLedEvents").html(`Student-Led Services (${studentLedEventsCount})`) : $("#studentLedEvents").html(`Student-Led Services`)
+      studentLedEventsCount > 0 ? $("#studentLedEvents").html(`Volunteer Programs (${studentLedEventsCount})`) : $("#studentLedEvents").html(`Volunteer Programs`)
```

Comment:
This change is not fully reflected. In html, it still is called Student Led services. So when I reload page, it first shows me "Student-Led Services" and only atfer a moment, it would show a new name. Fix that. Also, double check with Brian if we want to rename all the variables in all the files associated with student-led services. That variable is in a lot of places